coldfront/core/publication/views.py

```diff
@@ -247,80 +246,61 @@ def post(self, request, *args, **kwargs):
 
         return HttpResponseRedirect(reverse('project-detail', kwargs={'pk': project_pk}))
 
-
-class PublicationAddManuallyView(LoginRequiredMixin, UserPassesTestMixin, TemplateView):
+class PublicationAddManuallyView(LoginRequiredMixin, UserPassesTestMixin, FormView):
+    form_class = PublicationAddForm
     template_name = 'publication/publication_add_publication_manually.html'
 
     def test_func(self):
         """ UserPassesTestMixin Tests"""
         if self.request.user.is_superuser:
             return True
 
-        project_obj = get_object_or_404(
-            Project, pk=self.kwargs.get('project_pk'))
+        project_obj = get_object_or_404(Project, pk=self.kwargs.get('project_pk'))
 
         if project_obj.pi == self.request.user:
             return True
 
         if project_obj.projectuser_set.filter(user=self.request.user, role__name='Manager', status__name='Active').exists():
             return True
 
+        messages.error(self.request, 'You do not have permission to add a new publication to this project.')
+
```

Comment:
I see this error in the django admin interface, if I'm using the 'su' option to login as the user.  However, if I login directly as the user, this results in a 403 error with no explanation.

Reply:
Added 403 template so that those errors are now displayed.